gpflow/likelihoods/likelihoods.py

```diff
@@ -69,92 +69,307 @@ def inv_probit(x):
 
 
 class Likelihood(Module):
```

Comment:
```suggestion
class Likelihood(Module, metaclass=abc.ABCMeta):
```
and add `import abc` at the top

Reply:
done